src/backend/revoker/subScan.test.ts

```diff
@@ -84,52 +86,6 @@ describe('subScan', () => {
       expect(cTypeEvents.count).toBe(0);
       expect(cTypeEvents.events).toBeUndefined();
     });
-
-    it('should return parsed events in reverse order', async () => {
```

Comment:
The order of the events are specified by the parameter `order` on the request. So the api takes care of it. 

While mocking the api responses this will always fail. 

Maybe a new test could be made making sure that events are in chronological ascending order, but again, if we mock the api responses, it does not make so much sense. 

Maybe an assert statement can be included on `getEvents` and a new test could expect it to throw while the api responses are in the wrong order.

Reply:
Oh, I missed this handy parameter :+1: